packages/react-charting/src/components/GroupedVerticalBarChart/GroupedVerticalBarChart.types.tsx

```diff
@@ -80,6 +80,12 @@ export interface IGroupedVerticalBarChartProps extends ICartesianChartProps {
    * @default false
    */
   isCalloutForStack?: boolean;
+
+  /**
+   * Prop to hide the bar labels
+   * @default false
```

Comment:
Check that this doesn't affect any existing svg size.

Reply:
The SVG size is same as before. The text is already inside the SVG, so I haven't added any extra padding around the chart.